app/exporters/shopify/product_exporter.rb

```diff
@@ -1,48 +1,62 @@
 module Shopify
   class ProductExporter
-    include Spree::Retail::PresenterHelper
-
-    def initialize(spree_product_id:, product_klass: Spree::Product,
+    def initialize(spree_product:,
                    product_api: ShopifyAPI::Product,
-                   product_converter: Shopify::ProductConverter)
+                   product_attributes: Shopify::ProductAttributes)
 
-      @spree_product = product_klass.find(spree_product_id)
-      @product_converter = product_converter
+      @spree_product = spree_product
       @product_api = product_api
+      @product_attributes = product_attributes
     end
 
-    def perform
+    def save_product_on_shopify
```

Comment:
I agree this reads better but it locks us into the concept of Shopify. I know, I know there is more use of it throughout the gem but we need to take an active move to maybe remove it. Since this is quite a large factory style class (sure it doesn't create other classes here but it does in Shopify), `perform` or `call` seem like a cleaner interface, no?